### PR TITLE
PR3 - MD + Other

### PR DESCRIPTION
**Pull request by AntimatterReactor**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,10 +7,10 @@ matrix:
         - sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
         - sudo apt-get update -q
         - sudo apt-get install -y g++-7
-        - export CXX=/usr/bin/g++-7
+        - export CXX="/usr/bin/g++-7"
```
Comment (DynamicSquid) [inline .travis.yml:10]: this isn't the problem. it's something to do with the cmake command

**Review by DynamicSquid — CHANGES_REQUESTED** [review]
sweet! could you just remove the changes you did to the `.travis.yml` file, and then I can merge this!

**Comment by AntimatterReactor** [comment]
ok

**Comment by AntimatterReactor** [comment]
done!

**Comment by DynamicSquid** [comment]
and the quotations around `matrix`?

**Comment by DynamicSquid** [comment]
I'm getting off rn. I'll be back tmr morning to merge this PR, and then I'll take a look at the CSS/JS PR

**Comment by AntimatterReactor** [comment]
> 
> 
> and the quotations around `matrix`?

isn't it like the same before and after?

**Pull request merged**